aten/src/TH/generic/THTensorMoreMath.cpp

```diff
@@ -1161,7 +1160,14 @@ LAB_IMPLEMENT_VECTORIZED_FUNCTION(sigmoid,TH_MATH_NAME(TH_sigmoid),HYPER_TH_OMP_
 
 void THTensor_(polygamma)(THTensor *r_, int64_t n, THTensor *t) {
   switch (n) {
-    case 0: THTensor_(digamma)(r_, t); break;
+    case 0: {
+      THTensor_(resizeAs)(r_, t);
+      at::Tensor r__wrap = THTensor_wrap(r_);
+      at::Tensor t__wrap = THTensor_wrap(t);
+      at::Tensor result = at::native::digamma(t__wrap);
```

Comment:
You can use `_out` function and avoid additional copy (bet benchmarks showing slow down because of it)